.mage/js.go

```diff
@@ -157,6 +155,7 @@ func (js Js) Deps() error {
 		if err != nil {
 			return err
 		}
+		mg.Deps(JsSDK.Build)
```

Comment:
Remove

Reply:
This is necessary for production builds, when the SDK is not linked, as explained in https://github.com/TheThingsNetwork/lorawan-stack/pull/2764#discussion_r442224428